ngraph/core/src/op/variadic_split.cpp

```diff
@@ -188,32 +185,16 @@ namespace
                                   " during evaluate Split:v1");
             break;
         }
+
+        int64_t axis = host_tensor_2_vector<int64_t>(axis_tensor)[0];
+
         axis = ngraph::normalize_axis(split_node, axis, data_tensor->get_partial_shape().rank());
 
-        std::vector<int64_t> split_lengths;
         switch (split_lengths_tensor->get_element_type())
```

Comment:
The same for `split_lengths_tensor`

Reply:
I see that variadic_split has evaluate() so binary size concern is valid. Why we want to add more types if it was done on purpose?